src/server/tran_page_requests.hpp

```diff
@@ -56,5 +56,26 @@ enum class page_to_tran_request
   SEND_TO_PTS_LOG_PRIOR_LIST, /* response-less */
 };
 
+// requets from page server to page server to catchup
+enum class follower_to_followee_request
+{
+  // Reserve for responses
+  RESPOND,
+
+  SEND_DUMMY, // TODO for test. remove it.
+
+  // TODO SEND_DISCONNECT, /* response-less */
+  // TODO SEND_LOG_PAGES_FETCH, /* synchronously waiting for response */
```

Comment:
Question about design approach. I think it is nature that the followee(as client) request to follower(as server) to get some log page to catch-up. However, the implementation seems to do the opposite. Is there a reason the follower is designed to send requests to the followee?

Reply:
I guess you got confused. The Followee is the target to be caught up with. It means it's like a server and serves log pages to the Follower.

Reply:
OK. I resume the review.